proposals/20210818-driver-semver.md

```diff
@@ -18,7 +18,7 @@ not directly related to libscap itself. E.g. two different consumer releases can
 the same libscap commit and still be unable to share the probes.
 
 2. For the kernel module, there is no version check at all. The driver exposes an ioctl
-to get the probe version (`PPM_IOCTL_GET_PROBE_VERSION`) but it is not used anywhere.
+to get the probe version (`PPM_IOCTL_GET_DRIVER_VERSION`) but it is not used anywhere.
```

Comment:
```suggestion
to get the module version (`PPM_IOCTL_GET_DRIVER_VERSION`) but it is not used anywhere.
```

Reply:
Good catch. However, I prefer to fix that in another PR. I have also reverted the other change in the proposal files.